components/icon-button.jsx

```diff
@@ -4,18 +4,23 @@ var Link = Router.Link;
 var ImageTag = require('./imagetag.jsx');
 
 var LinkSwap = React.createClass({
+  propTypes: {
+    linkTo: React.PropTypes.string,
+    href: React.PropTypes.string,
+    onClick: React.PropTypes.func
```

Comment:
@toolness I'm not sure if `<Link>` takes any valid `<a>` attributes.  It seems like it doesn't... ? Mind double checking for me?

- https://github.com/rackt/react-router/blob/master/modules/Link.js#L77-L80
- https://github.com/rackt/react-router/blob/57543eb41ce45b994a29792d77c86cc10b51eac9/build/lib/createRouter.js#L188-L214